domains/auth/Auth.API/Integration.Tests/Controllers/LogoutControllerTests.cs

```diff
@@ -48,37 +54,31 @@ public async Task LogoutAsync_ShouldReturnErrorCodeUrl_WhenDiscoveryCacheFails()
         var client = factory.CreateAuthenticatedClient(user);
 
         var result = await client.GetAsync("auth/logout");
+        var resultContent = await result.Content.ReadAsStringAsync();
+        var redirectUriResponse = JsonConvert.DeserializeObject<RedirectUriResponse>(resultContent);
         Assert.NotNull(result);
```

Comment:
This `assert` should happen before we use the variable. Also the very next thing should be the HTTP status code OK assert.